pkg/backend/kubeflow/generator/cm.go

```diff
@@ -0,0 +1,213 @@
+package generator
+
+import (
+	"fmt"
+
+	s2iconfigmap "github.com/caicloud/ciao/pkg/s2i/configmap"
+	pytorchv1alpha2 "github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1alpha2"
+	tfv1alpha2 "github.com/kubeflow/tf-operator/pkg/apis/tensorflow/v1alpha2"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/caicloud/ciao/pkg/types"
+)
+
+const (
+	baseImageTF      = "tensorflow/tensorflow:1.10.1-py3"
```

Comment:
any thoughts on making this configurable?

Reply:
Ref #35 

Not very clear now.